### ISSUE TITLE
Resource avatar on project task card accessibility issue

### ISSUE THREAD

**Issue by chuckn0rris** [bryntum/support#11181]
[Forum post](https://forum.bryntum.com/viewtopic.php?f=51&t=32685&p=166895#p166895)

We're facing a few UX issues with the avatars.

In Task Board, our UX designers are not happy with the color contrast of the numeric overlay on the opaque background. The accessibility tools in Firefox say it is a 3.55:1 ratio which violates the "1.4.3 Contrast (Minimum)" WCAG 2.1 rule. The problem is made worse when resource initials are used, as it is text overlaying text, which is difficult to read.

As such, we have applied a solid background to the element which makes the avatar on the element fully hidden. However, the count now appears incorrect, e.g. it says +2 but there are 3 resources in the overflow tooltip.

I think the underling issue here is the dual meaning of the last element. It represents both a resource assignment and any resources in the overflow simultaneously. Would it be possible to change the design and implementation so the last element does not represent a resource and instead represents the overflow group as a whole?

Separately, there is an issue with the avatar overflow in Gantt. The resource on the overflow element isn't being read out by screen readers as it isn't part of the aria-label for the "b-overflow-img" div. Perhaps this too could be fixed by using the final element as an overflow symbol instead of an actual resource.
